performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -58,6 +115,7 @@ class AnalysisResult
     const StatisticsTracker sub_loop_time_reserve
   );
 
+  AnalysisResult() {}
```

Comment:
Is the default constructor required only when ODB is enabled or always? If only for ODB then can be put in ifdefs

Reply:
done

Reply:
Okay great. Thanks